src/Servant/Job/Core.hs

```diff
@@ -183,10 +185,10 @@ newID :: KnownSymbol k => Proxy k -> SecretKey -> UTCTime -> Int -> ID 'Safe k
 newID p s t n = PrivateID tn n t $ macID tn s t n
   where tn = symbolVal p
 
-mkID :: KnownSymbol k => Proxy k -> Int -> UTCTime -> String -> ID 'Unsafe k
+mkID :: KnownSymbol k => Proxy k -> Int -> UTCTime -> String -> ID safety k
 mkID p n t d = PrivateID (symbolVal p) n t d
 
-instance (KnownSymbol k, safety ~ 'Unsafe) => FromHttpApiData (ID safety k) where
+instance (KnownSymbol k{- , safety ~ 'Unsafe -}) => FromHttpApiData (ID safety k) where
```

Comment:
The original author left these in, possibly mostly as commentary, but I can clean them up.